examples/StandaloneMain/hellovr_opengl_main.cpp

```diff
@@ -596,12 +596,12 @@ bool CMainApplication::BInit()
 {
 	b3Printf( "GL Error: %s\n", message );
 }
-*/
 
 static void APIENTRY DebugCallback (GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length, const GLchar* message, GLvoid* userParam)
```

Comment:
This was complaining about Gsizei. As far as I cant tell it shouldn't be.

https://gist.github.com/BlGene/f062ee5166ff0acb02a5403657d2c057